packages/edit-site/src/components/dataviews/in-filter.js

```diff
@@ -9,10 +9,13 @@ import {
 const OPERATOR_IN = 'in';
 
 export default ( { filter, view, onChangeView } ) => {
-	const activeValue = view.filters.find(
+	let activeValue = view.filters.find(
 		( f ) => f.field === filter.id && f.operator === OPERATOR_IN
 	)?.value;
 
+	if ( ! activeValue ) {
```

Comment:
Not affecting us right now, but this would make zero value impossible to set.

Reply:
you're right maybe we can make a small change here and drop the `?.value;` from the line above and only do `.value` in the rendering of the component.

Reply:
Nice one, thanks! Pushed a fix.